packages/analytics-types/src/amplitude-promise.ts

```diff
@@ -1,3 +1,3 @@
 export interface AmplitudeReturn<T> {
-  promise: Promise<T>;
+  promise: T;
```

Comment:
Seems a little weird the promise is no longer a Promise. It is possible to keep Promise<T> on this interface?

Reply:
I agree! I tried different variations and this is the best I could get to without using `Awaited` :\